AntennaTracker/Parameters.cpp

```diff
@@ -379,6 +379,14 @@ const AP_Param::Info Tracker::var_info[] = {
     // @Increment: 0.5
     // @User: Advanced
 
+    // @Param: PITCH2SRV_PDMX
+    // @DisplayName: Pitch axis PD Sum Maximum
+    // @Description: The maximum/minimum value that the sum of the P and D term can output
+    // @Range: 0 4000
+    // @Increment: 10
+    // @Units: d%
```

Comment:
Is this correct the units? You don't have it everywhere and it looks a little weird to me

Reply:
Yeh, they are using centidegrees.